projects/plugins/jetpack/extensions/blocks/contact-form/child-blocks.js

```diff
@@ -50,6 +50,38 @@ const FieldDefaults = {
 			type: 'number',
 			default: 100,
 		},
+		borderRadius: {
+			type: 'number',
+			default: 0,
+		},
+		borderWidth: {
+			type: 'number',
+			default: 0,
```

Comment:
I'd say this should be (at least) 1? Here's what the form looks like when just inserted:

Editor:
<img width="752" alt="image" src="https://user-images.githubusercontent.com/157240/212983647-a4a86305-4a00-4acf-a383-0356ec26f686.png">

Yet it looks ok on publish:
<img width="785" alt="image" src="https://user-images.githubusercontent.com/157240/212983786-f2d8b488-4030-48f8-aea9-2cb8ae34cde7.png">

Reply:
You're right. That should be 1, but if the default is set set to 1 and I clear the setting, it will be set to 1 when I reload the page. If I set it to 0 then it will at least stay at 0. I guess it's something that users could get used to.

Reply:
The default is now "" and when the setting is set to "" that bit of CSS isn't added to the input. It will be the theme that decides how the input is displayed.